lakefile.lean

```diff
@@ -1,12 +1,10 @@
 import Lake
 open Lake DSL System
 
-package proofwidgets {
+package proofwidgets where
   preferReleaseBuild := true
-  buildDir := "build"
```

Comment:
@digama0 we have just set this in #32 precisely so that we wouldn't have to make global changes to the build location.

Reply:
I think that it is important for the build location to match the default unless there is a good reason to be non-default. This is causing problems for cache, which does not yet have the ability to read lakefiles because lake hasn't exposed the API.

Reply:
I see, okay then!